core/server/services/slack.js

```diff
@@ -48,11 +54,51 @@ function ping(post) {
             return;
         }
 
+        var blogUrl = config.get('url');
+        var blogTitle = settingsCache.get('title');
```

Comment:
Any new code should use `let` or `const`.

Reply:
Will change that. I thought the code in the same file was using es5, that's why I used `var`.